mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -50,3 +66,4 @@ CLASS net/minecraft/client/Minecraft
 	METHOD method_3332 isFancyGraphicsEnabled ()Z
 	METHOD method_3333 isAmbientOcclusionEnabled ()Z
 	METHOD method_3334 isDebugEnabled ()Z
+	METHOD method_3336 getBlockInfo ()Lnet/minecraft/class_245;
```

Comment:
As is above, this is an OS type enum used for appdata directories and such, definitely not block info stuff. Unmap this one